updatemanager.go

```diff
@@ -110,28 +110,28 @@ func newUpdateManager(cfg *config.Config) (um *updateManager, err error) {
 		}
```

Comment:
Fix commit message: instanse -> instance, perfoem -> perform. Do not break the message body, it fits 80 symbols. Put period at the end of message body.